typings/balena-sdk.d.ts

```diff
@@ -463,6 +466,7 @@ declare namespace BalenaSdk {
 		vpn_address: string | null;
 		should_be_managed_by__supervisor_release: number;
 		api_heartbeat_state: 'online' | 'offline' | 'timeout' | 'unknown';
+		overall_status: DeviceOverallStatus.OverallStatus;
```

Comment:
I would love to have a `/** */` comment that this is computed, or maybe even try declaring it as `readonly`!

Reply:
I don't think the `readonly` is necessary, otherwise we need to set it to all of them (since ideally we never modify the object)

Reply:
It isn't necessary indeed, I was just thinking of a way to represent that this is a computed term in a way the TS can understand it (and in the future error if you try to patch it).